solc/CommandLineInterface.cpp

```diff
@@ -1184,6 +1206,16 @@ bool CommandLineInterface::processInput()
 
 		return assemble(inputLanguage, targetMachine, optimize, yulOptimiserSteps);
 	}
+	else
+	{
```

Comment:
Brackets can be removed (if there's no intention to add more things inside the `else`)

Reply:
Done.